src/components/Picklist/readme.md

```diff
@@ -401,18 +401,19 @@ function handlePicklistChange(value) {
                 label="Building Location"
                 required
                 onChange={value => setState({ value })}
-                value={state.value}
+                value={state.googleAddressValue}
```

Comment:
All the Inputs of the lib are controlled components, to use them you have to pass the value and update the value in the onChange.